components/TextFields/src/MDCTextInputControllerBase.h

```diff
@@ -59,6 +59,13 @@ extern const CGFloat MDCTextInputControllerBaseDefaultBorderRadius;
 */
 @interface MDCTextInputControllerBase : NSObject <MDCTextInputControllerFloatingPlaceholder>
 
+/**
+ The color behind the input and label that defines the preferred tap zone.
+
+ Default is borderFillColorDefault.
+ */
+@property(nonatomic, nullable, strong) UIColor *backgroundColor;
```

Comment:
I think this is null_resettable too since it pulls the default if it's nil and the default is null_resettable.